### PR TITLE
Simsingh/cert manager

### PR DESCRIPTION
**Pull request by simardeep1792**
Changes:
- Removed the Chainguard Images for the cert-manager
- Updated the CRDs for the cert-manager
- Removed certificate.yaml from the cert-manager folder
- Refactored the let's encrypt certificate from Issuer to Cluster Issuer

### CODE REVIEW

File: k8s/cert-manager/kustomization.yaml [hunk 1]
```diff
@@ -2,4 +2,29 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
 resources:
   - issuers.yaml
-  - certificate.yaml
+patches:     
+- target:
+    group: cert-manager.io
+    version: v1
+    kind: ClusterIssuer
+    name: letsencrypt
+    namespace: istio-system
```
Comment (vedantthapa) [inline k8s/cert-manager/kustomization.yaml:11]: Since it's a `ClusterIssuer`, I don't think we'll be needing this.

**Review by vedantthapa — CHANGES_REQUESTED** [review]

File: k8s/cert-manager/kustomization.yaml (hunk 1 above)
Comment (simardeep1792) [reply k8s/cert-manager/kustomization.yaml]: Good catch! fixed and pushed

**Review by Stephen-ONeil — APPROVED** [review]
:+1:

**Comment by vedantthapa** [comment]
@simardeep1792 if things are all set here can we merge this?

**Pull request merged**